windows_containers/configuring-byoh-windows-instance.adoc

```diff
@@ -0,0 +1,65 @@
+[id="configuring-byoh-windows-instance"]
+= Configuring Bring-Your-Own-Host (BYOH) Windows instances
+
+-NEED AN INTRO-
+
+== Prerequisites
+
+Any Windows instances that are to be attached to the cluster as a node must fulfill the following requirements:
+
+* The Docker container runtime must be installed on the instance.
+* The instance must be on the same network as the Linux worker nodes in the cluster.
+* Port 22 must be open and running an SSH server.
+* Port 10250 must be open for log collection.
+* An administrator user is present with the private key used in the secret set as an authorized SSH key.
+* The hostname of the instance must follow the link:https://datatracker.ietf.org/doc/html/rfc1123[RFC 1123] DNS label standards, which include:
+** Contain only lowercase alphanumeric characters or '-'.
+** Start with an alphanumeric character.
+** End with an alphanumeric character.
+
+== Procedure
+. Create a ConfigMap named `windows-instances` in the WMCO namespace that describes the Windows instances to be added.
++
+[NOTE]
+====
+Each entry in the data section of the ConfigMap should be formatted with the address as the key and a value with the
```

Comment:
- s/ConfigMap/config ma
- You might also want to consider switching to active voice, such as something like:

> Format each entry in the config map's data section by using the address as the key while formatting the value as `username=<username>`.